pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/EasyPostUtil.java

```diff
@@ -307,45 +309,29 @@ public Shipment getShipment(String shipmentId) throws EasyPostException {
      * @return true if easypost was able to create the address, false otherwise
      */
 
-    public boolean checkAddress(@NonNull JuniperKitRequest juniperKitRequest, String phone) {
-        if ((StringUtils.isBlank(juniperKitRequest.getJuniperParticipantID()))
-                || StringUtils.isBlank(juniperKitRequest.getLastName())) {
-            return false;
+    public String getEasyPostAddressId(@NonNull JuniperKitRequest juniperKitRequest, String phone, DeliveryAddress deliveryAddress)
```

Comment:
Changed the name of the method and it's return ro String so it is easier to test